scipy/special/_basic.py

```diff
@@ -14,7 +14,7 @@
 from . import _ufuncs as ufuncs
 from ._ufuncs import (mathieu_a, mathieu_b, iv, jv, gamma,
                       psi, hankel1, hankel2, yv, kv, ndtri,
-                      poch, binom, hyp0f1)
+                      poch, binom, hyp0f1, _factorial)
```

Comment:
p.s. just use the above `ufuncs` import and do `ufuncs._factorial`. Generally trying to cut down on the number of un-namespaced functions in `_basic.py`.

Reply:
ah yeah I see your point. I hadn't considered it would be in there now with the import-changed.